libraries/tools/kotlin-stdlib-gen/src/templates/Elements.kt

```diff
@@ -811,6 +811,10 @@ object Elements : TemplateGroupBase() {
         includeDefault()
         include(Lists, CharSequences, ArraysOfUnsigned)
     } builder {
+        specialFor(CharSequences) {
```

Comment:
This whole block should be placed after the original "sample" block, otherwise, the latter will override `sample("samples.text.Strings.findLast")`.

Reply:
I could understand your comment. Update moving after original `sample` method: https://github.com/JetBrains/kotlin/pull/4871/commits/bce231a8be4dc97e07424b3635b708a11cf25c85